packages/nocodb/src/cache/NocoCache.ts

```
@@ -129,6 +129,71 @@ export default class NocoCache {
+  ): Promise<boolean> {
+    if (this.cacheDisabled) return Promise.resolve(true);
+    if (Object.keys(hash).length === 0) {
+      return;
+    }
+    return !!this.client.setHash(`${this.prefix}:${key}`, hash, options);
+  }
+
+  public static async getHash(
+    key: string,
+  ): Promise<Record<string, string | number>> {
+    if (this.cacheDisabled) return Promise.resolve({});
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)